libs/designer/src/lib/core/actions/bjsworkflow/initialize.ts

```diff
@@ -248,6 +248,10 @@ export const getOutputParametersFromManifest = (
   };
 };
 
+export const getParsedManifest = (manifest: OperationManifest): ManifestParser => {
```

Comment:
why make this a function? seems redundant unless you plan on adding more functionality.